api/definitions.yaml

```diff
@@ -336,6 +339,11 @@ IsInboxEnabled:
   description: |-
     True if the recipient of a message wants to store its content for
     later retrieval.
+IsTosAccepted:
+  type: boolean
```

Comment:
we need to store the version (a number) of the last ToS accepted, the app will check this number against the latest ToS and if there's a newer ToS will ask the user to accept again